vulture/core.py

```diff
@@ -461,7 +449,7 @@ def is_identifier(name):
                 self.used_names.add(name)
             for attr in name_and_attrs[1:]:
                 if is_identifier(attr):
-                    self.used_attrs.add(attr)
+                    self.used_names.add(attr)
```

Comment:
The code for the name and the attributes can be combined now.

Reply:
done!